<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-SR9OPSDN" ignored="0" name="Login Test" tests="2" failures="1" timestamp="02 May 2022 02:19:21 GMT" time="8.754" errors="0">
  <testcase name="verifyLogin" time="3.155" classname="datadriven_testCases.TC001_LoginDDT">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/index.php/dashboard] but found [https://opensource-demo.orangehrmlive.com/index.php/auth/validateCredentials]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/index.php/dashboard] but found [https://opensource-demo.orangehrmlive.com/index.php/auth/validateCredentials]
	at datadriven_testCases.TC001_LoginDDT.verifyLogin(TC001_LoginDDT.java:25)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogin -->
  <testcase name="verifyLogin" time="2.825" classname="datadriven_testCases.TC001_LoginDDT"/>
</testsuite> <!-- Login Test -->
